src/app/(BottomTabNavigation)/Profile/LogOut/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { router } from 'expo-router';
+import React from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import SignOut from '../../../../../assets/sign-out.svg';
+import X from '../../../../../assets/x.svg';
+// import CasesHeader from '../../../../Components/CasesHeader/CasesHeader'; address in future PR
+import { useSession } from '../../../../context/AuthContext';
+function LogOutConfirmation() {
+  const { signOut, session } = useSession();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.line}>
+        <View style={styles.screenContainer}>
```

Comment:
We only need one of these two containers. We should be able to merge their props.